### PR TITLE
changes

### PR DESCRIPTION
**Pull request by celineaudinsap**

### CODE REVIEW

**Review by celineaudinsap — COMMENTED** [review]
Added important note

**Review by celineaudinsap — COMMENTED** [review]
important note

File: tutorials/irpa-setup-1-booster-subscription/irpa-setup-1-booster-subscription.md [hunk 1]
```diff
@@ -8,20 +8,27 @@ tags: [ tutorial>beginner, software-product>sap-business-technology-platform]
 primary_tag: software-product>sap-intelligent-robotic-process-automation
 ---
 
+
 # Subscribe to SAP Intelligent RPA Service in SAP BTP
 <!-- description --> Subscribe to SAP Intelligent RPA on SAP Business Technology Platform so that you can build an automation to automate repeated tasks.
 
+> **IMPORTANT:** Note: [The SAP BTP trial offering will no longer be available as of January 31st, 2023](https://blogs.sap.com/2023/01/09/trial-accounts-retirement-of-sap-intelligent-rpa-end-of-january-2023-and-the-move-forward/). Please move to an SAP Free Tier account and subscribe to SAP Process Automation.
```
Comment (MichaelCzcz) [inline tutorials/irpa-setup-1-booster-subscription/irpa-setup-1-booster-subscription.md:15]: Please remove the word "Note".

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]
I am not sure about the primary tag software-product>sap-build and I could not find it in IMS.
@jmmargo can you confirm that this tag does exist?

**Pull request closed without merging**